src/gl_context.rs

```diff
@@ -47,8 +48,10 @@ impl<Native> GLContext<Native>
         };
 
         try!(native_context.make_current());
+        let extensions = gl_.get_string(gl::EXTENSIONS);
+        let extensions: Vec<String> = extensions.split(&[',',' '][..]).map(|s| s.into()).collect();
```

Comment:
Oh, didn't notice you were saving all the extensions there, right. Then that looks fine.